java/README.md

```diff
@@ -0,0 +1,44 @@
+# Summary
+The Java layer for Vowpal Wabbit uses the [Java Native Interface](https://en.wikipedia.org/wiki/Java_Native_Interface) to communicate between Java and the native Vowpal Wabbit C code.  The philosophy of this layer is that we want to expose a simple type safe, thread safe interface so a Java programmer can use Vowpal Wabbit for either learning or scoring.  An example usage is shown below
+
+```java
+class Foo {
+    static {
+        VWScalarLearner learner = VWLearners.create("-f model.vw"); // Create the model.  The model type is checked here
```

Comment:
I would put the comments on top of the code instead of to the right and add double new lines between 

* model creation
* learning
* closing